test/groovy/JenkinsMaterializeLogTest.groovy

```diff
@@ -1,4 +1,9 @@
 import hudson.FilePath
+import hudson.console.AnnotatedLargeText
```

Comment:
I guess there are some un-used imports.
Could you please remove them? Thanks!

Reply:
Saw this too late...